ui/apps/platform/cypress/integration/vulnerabilities/nodeCves/cveDetailPage.test.ts

```diff
@@ -0,0 +1,73 @@
+import withAuth from '../../../helpers/basicAuth';
+import { hasFeatureFlag } from '../../../helpers/features';
+
+describe('Node CVEs - CVE Detail Page', () => {
+    withAuth();
+
+    before(function () {
+        if (!hasFeatureFlag('ROX_VULN_MGMT_NODE_PLATFORM_CVES')) {
+            this.skip();
+        }
+    });
+
+    it('should restrict access to users with insufficient permissions', () => {
+        // check that users without Node access cannot access the Node CVE Detail page directly
+    });
+
+    it('should only show relevant filters for the Node CVE Detail page', () => {
```

Comment:
A thought about descriptions. **Node CVE Detail page** for test seems redundant with description of `describe` block.
* That is, less might be more here, similar to the title of the preceding test.
* However, **Node** table in tests below seems like natural repetition.